compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorListener.java

```diff
@@ -47,7 +45,8 @@ public void reportInvalidNodeError(STToken token, String message) {
     }
     
     private void logError(int line, int col, String message) {
-        PrintStream out = System.out;
-        out.println("xxx.bal:" + line + ":" + col + ":" + message);
+        // TODO - Removing below lines as it breaks lang-server rpc
```

Comment:
We are fixing this with the following PR. 

https://github.com/ballerina-platform/ballerina-lang/pull/23500

Reply:
Noted